openstack/containerinfra/v1/clusters/requests.go

```diff
@@ -175,3 +175,40 @@ func Update(client *gophercloud.ServiceClient, id string, opts []UpdateOptsBuild
 	}
 	return
 }
+
+// ResizeOptsBuilder allows extensions to add additional parameters to the
+// Resize request.
+type ResizeOptsBuilder interface {
+	ToClusterResizeMap() (map[string]interface{}, error)
+}
+
+// ResizeOpts params
+type ResizeOpts struct {
+	NodeCount     *int     `json:"node_count" required:"true"`
+	NodesToRemove []string `json:"discovery_url,omitempty"`
```

Comment:
It seems that it should have the `json:"nodes_to_remove,omitempty"` tag given changes from [#638572](https://review.openstack.org/#/c/638572/22/magnum/api/controllers/v1/cluster_actions.py)

Reply:
Maybe it's also better to provider an additional unit test for this parameter?

Reply:
> It seems that it should have the json:"nodes_to_remove,omitempty"

Good catch - thank you!

Reply:
@ozerovandrei  Yes, the NodesToRemove is also optional. Thanks.

Reply:
@openstacker Just to clarify, @ozerovandrei meant that the JSON tag `discovery_url` does not match `NodesToRemove`. `discovery_url` needs to be changed to `nodes_to_remove`.

Reply:
Ah, yes, sorry, that's a copy-paste issue. I just fixed it. Thanks.